lib/src/kinde_flutter_sdk.dart

```diff
@@ -297,13 +297,13 @@ class KindeFlutterSDK with TokenUtils, HandleNetworkMixin {
     }
   }
 
-  Uri _buildEndSessionUrl() {
-    var uri = Uri.parse(_serviceConfiguration.endSessionEndpoint!)
+  WebUri _buildEndSessionUrl() {
+    var uri = WebUri(_serviceConfiguration.endSessionEndpoint!)
         .replace(queryParameters: {
       _postLogoutRedirectParamName: _config!.logoutRedirectUri,
```

Comment:
the same value `_config!.logoutRedirectUri`
is set to both _postLogoutRedirectParamName and _redirectParamName.

is this expected?